### DISCUSSION TITLE
Unable to display file structure

### DISCUSSION THREAD

**Discussion by ethereall996** [community/community#86697] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I access the project repository, the file structure of the code cannot be displayed.
Under the same network conditions, other devices can access it normally, but the computer cannot. It has been confirmed that there is no problem with the host file on the computer. What other possible reasons are there for seeking help?   help！help！

**Comment by ethereall996** [comment]
like this：
![image](https://github.com/community/community/assets/104899725/abeea8ce-c74e-4ff9-9d65-5869943afffe)

**Comment by ghost** [comment]
i guess this would help:

- Ensure that your computer is connected to the network properly.
Try accessing other websites or repositories to verify that your internet connection is stable.
- Clear your browser cache and cookies. Sometimes, outdated or corrupted cache files can cause issues with accessing websites.
- Try accessing the repository using a different web browser to see if the issue is browser-specific.
- Confirm that the repository URL is correct and hasn't changed. Verify with other team members or the repository owner.

thank you

**Comment by mewmew** [comment]
From https://github.com/orgs/community/discussions/87019 (reported by @RJVB)

> ### Select Topic Area
> 
> Bug
> ### Body
> 
> Hello,
> 
> I think there has been a regression in the fancy file-tree display code that breaks functionality on older browsers, and thus for people that are still using older OS versions. I am hit by this on my Mac because it is limited to Firefox 78.15 and I think there is no good justification for disallowing the use of "such an old" browser. The machine is still perfectly valid and fast enough for the software development I do and the github.com website used to work perfectly fine until recently (all the other parts still seem to do, in fact).
> 
> I think ticket #86697 describes the same issue.
> 
> ![Screen Shot 2024-01-13 at 22 32 12](https://private-user-images.githubusercontent.com/1756884/296516466-53928ef4-8ccd-4a3e-8e2d-6012dd3f4c54.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMzNTczNTksIm5iZiI6MTcxMzM1NzA1OSwicGF0aCI6Ii8xNzU2ODg0LzI5NjUxNjQ2Ni01MzkyOGVmNC04Y2NkLTRhM2UtOGUyZC02MDEyZGQzZjRjNTQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDQxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA0MTdUMTIzMDU5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9M2I3MTMwOWM1YWRkNDVmMTBkZGNlZjZmMmIyZWMzMDJlYTk1Zjc1NDlhMjVlZmRlMjc1MDM5MGE3NjQ3ZjJmYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.rZ3age2F3_7yuoSkvNV7cHkAO9SPW46e4Uf7ZX2yrik)
> 
> I _can_ use "Chromium Legacy" which is a current-version build of Chromium patched for legacy systems. That browser is not stable enough to be my main browser but it is not affected by this issue. That confirms that the issue at hand is indeed a regression in github code and not a problem on my system.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @mewmew !

I didn't catch that the original post was in the wrong category! I'm gonna move this over to Code Search and Navigation, as its an issue with Code View Navigation; hopefully it will get more traction with the community over there. 

Also 

This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions! ❤️ 🚀
